python/mxnet/cython/ndarray.pyx

```diff
@@ -52,7 +52,7 @@ cdef class NDArrayBase:
         def __get__(self):
             return bool(self.cwritable)
 
-    def __init__(self, handle, writable=True):
+    def __init__(self, handle, writable=True, stype=-1):
```

Comment:
Is stype required here?

Reply:
No, it is not required. I updated the code.